neat.h

```diff
@@ -6,6 +6,10 @@
 #include <sys/types.h>
 #include <netinet/in.h>
 
+#ifndef IPPROTO_SCTP
```

Comment:
Why do you need this? Up to now I #ifdef'ed out code needing this. usrsctp.h contains this, so there should not be need for this.